src/sc/sidechain.cpp

```diff
@@ -713,20 +713,27 @@ int CSidechain::getNumBlocksForScFeeCheck()
 
 int CSidechain::getMaxSizeOfScFeesContainers()
 {
-    int epochLength = fixedParams.withdrawalEpochLength;
-    int numBlocks = getNumBlocksForScFeeCheck();
-
-    //assert(epochLength > 0);
-    if (epochLength == 0) {
-        maxSizeOfScFeesContainers = 1;
-    } else {
-        maxSizeOfScFeesContainers = numBlocks / epochLength;
-        if (maxSizeOfScFeesContainers == 0 || (numBlocks % epochLength != 0) )
-        {
-            maxSizeOfScFeesContainers++;
+    if (maxSizeOfScFeesContainers == -1) {
+        if (!isNonCeasing()) {
+            const int numBlocks = getNumBlocksForScFeeCheck();
+            const int epochLength = fixedParams.withdrawalEpochLength;
+            assert(epochLength > 0);
+            maxSizeOfScFeesContainers = (numBlocks + epochLength - 1) / epochLength;
+            // CSidechain::getNumBlocksForScFeeCheck() may return 0 for regtest...
+            // This was managed in the same way in the old version
+            maxSizeOfScFeesContainers = std::max(maxSizeOfScFeesContainers, 1);
+        } else {
+            // As defined in chainparams.cpp, maxSizeOfScFeesContainers is 200 for mainnet and testnet,
```

Comment:
I guess `maxSizeOfScFeesContainers` should be `ScNumBlocksForScFeeCheck` (or `nScNumBlocksForScFeeCheck`)

Reply:
Comment updated, to better explain the origin of the value and avoid confusion, in c8b2941bc959c29cd0db69eb8df66aa425bccda8